packages/terra-clinical-data-grid/src/DataGrid.jsx

```diff
@@ -1084,7 +1089,12 @@ class DataGrid extends React.Component {
         >
           <div
             className={cx('vertical-overflow-container')}
-            ref={this.setVerticalOverflowContainerRef}
+            ref={(ref) => {
```

Comment:
@habelk, do these code change address your ask? 

> This request is to expose a ref callback prop on the DataGrid to expose the overflow container.

Reply:
We probably want to expose a ref to the 'horizontal-overflow-container' below as well. That will allow consumers to manipulate both the vertical and horizontal scrolling.

Reply:
Updated here https://github.com/cerner/terra-clinical/pull/585/commits/b069773ed557b1c21ab850266334bffe3dfcc245.

Reply:
Yes, this satisfies our needs for scroll positioning. I will log a separate issue for other clinical data grid related enhancements based on https://github.com/cerner/terra-clinical/issues/580#issuecomment-545954846